nurseconnect/forms.py

```diff
@@ -0,0 +1,44 @@
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+from molo.profiles.forms import EditProfileForm
+from molo.profiles.models import UserProfile
+
+
+class NurseConnectEditProfileForm(EditProfileForm):
```

Comment:
Oh so if I understand correctly, its because a ModelForm is best used when coupled with a model?